### PR TITLE
Updates the benchmark with inner iterations

### PR DESCRIPTION
**Pull request by nkcr**
- adds the WASM execution environment
- updates the benchmark to use the `b.N` iteration count
- adds documentation on how to run the benchmarks
- adds a plot

Screenshot 2022-01-26 at 16 12 09

### CODE REVIEW

File: goland/unikernel_net_fs_ec/mod.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	conn.SetReadDeadline(time.Now().Add(time.Second * 100))
+
+	_, err = conn.Read(readRes)
+	if err != nil {
+		return res, xerrors.Errorf("failed to read result: %v", err)
+	}
+
+	snap.Set(resultKey[:], readRes)
+
+	// resultBuf, err := hex.DecodeString(string(readRes))
```
Comment (jbsv) [inline goland/unikernel_net_fs_ec/mod.go:92]: still useful ?

Comment (pierluca) [reply goland/unikernel_net_fs_ec/mod.go:92]: I'd have to agree / expand, there's quite a lot of commented out code, in both Go (tcp_ec, unikernel_net_fs_ec, etc.) and WASM (`wasm_env/c/*.c`).
I'd consider removing it unless there's a reason to keep it around.

Comment (nkcr) [reply goland/unikernel_net_fs_ec/mod.go:92]: This code is useful. It shows the output format and can help debug the execution.

Comment (nkcr) [reply goland/unikernel_net_fs_ec/mod.go:92]: I understand your point, however I don't want to remove comments / commented code left by the students yet. Some comments and commented code are actually useful. I want to be sure I can follow what the student did and document the process before doing so.

Comment (jbsv) [reply goland/unikernel_net_fs_ec/mod.go:92]: Discussed with Noémien and agreed that having this legacy point in the version history has some value, even though we want to clean it up in a future version.

**Pull request merged**